src/meta/src/hummock/manager/mod.rs

```diff
@@ -501,12 +501,10 @@ impl HummockManager {
                 pairs.push((table_id as StateTableId, group.id));
             }
             let group_config = group.compaction_config.clone().unwrap();
-            self.compaction_group_manager
-                .write()
-                .await
-                .init_compaction_config_for_replay(group.id, group_config)
-                .await
-                .unwrap();
+            let mut compaction_group_manager = self.compaction_group_manager.write().await;
+            let insert_trx = compaction_group_manager.insert_config_trx(group.id, group_config);
+            commit_multi_var!(self.meta_store_ref(), insert_trx)?;
```

Comment:
Shall we commit all these transaction of every group together ?

Reply:
However, init_metadata_for_version_replay is only used in testing, which I think is acceptable.